src/app/app-routing.module.ts

```diff
@@ -31,6 +36,10 @@ const routes: Routes = [
         path: 'sample-questions',
         component: SampleQuestionsComponent
     },
+    {
+        path: 'user-stats',
+        component: UserStatsComponent
```

Comment:
This should not be here.
It should've been added in the the PR that creates user stats component.